gulpfile.js

```diff
@@ -80,6 +80,9 @@ gulp.task('package-js', function() {
         .pipe(gulp.dest('./'));
 });
 
+// Packages all static content
+gulp.task('package', ['package-content', 'package-css', 'package-js'], function() {})
+
 // Deploys the packaged files to the server. Requires Sublime SFTP to be set up in the project.
 gulp.task('deploy', ['package-content', 'package-css', 'package-js'], function() {
```

Comment:
Cool :) This could now depend on `package`, I'll send a small follow-up. Thanks!